docs/cluster_admin/activating_feature_gates.md

```diff
@@ -48,5 +48,5 @@ Change the namespace to suite your installation.
 
 ## List of feature gates
 The list of feature gates (which evolve in time) can be checked directly from
-the [source code](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-config/feature-gates.go#L26).
+the [source code](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-config/featuregate/active.go#L22) (use the const string values in the yaml configuration).
```

Comment:
Thank you very much @pyaillet :)

WDYT about removing the line number altogether?
```suggestion
the [source code](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-config/featuregate/active.go) (use the const string values in the yaml configuration).
```

The clarification you've added about using the const string is good enough in my opinion, as the line number will likely change again in the future. What do you think?

Reply:
Yes, of course!

I just applied the suggestion.